packages/mobile/src/exchange/saga.ts

```diff
@@ -36,19 +43,70 @@ const LARGE_DOLLARS_SELL_AMOUNT_IN_WEI = new BigNumber(1000 * 100000000000000000
 const LARGE_GOLD_SELL_AMOUNT_IN_WEI = new BigNumber(100 * 1000000000000000000)
 const EXCHANGE_DIFFERENCE_TOLERATED = 0.01 // Maximum difference between actual and displayed takerAmount
 
-export function* doFetchExchangeRate(makerAmount?: BigNumber, makerToken?: CURRENCY_ENUM) {
-  Logger.debug(TAG, 'Calling @doFetchExchangeRate')
+export function* doFetchTobinTax({ makerAmount, makerToken }: FetchTobinTaxAction) {
+  try {
+    let tobinTax
+    if (makerToken === CURRENCY_ENUM.GOLD) {
+      yield call(getConnectedAccount)
+
+      // Using native web3 contract wrapper since contractkit
+      // hasn't yet implemented tobin tax interface
+      const reserve = yield call([
+        contractKit._web3Contracts,
+        contractKit._web3Contracts.getReserve,
+      ])
+
+      const tobinTaxFraction = yield call(reserve.methods.getOrComputeTobinTax().call)
```

Comment:
Note that the Reserve contract lacks a view function for the Tobin tax so we use .call() that executes the transaction without sending it

Looked into how `web3.eth.contracts` handles method calls for a view function vs for a state-changing function that is only called- both are eventually resolved to just "calls", so we aren't making the node do more work this way. See implementation of functions on contracts here https://github.com/ethereum/web3.js/blob/v1.2.0/packages/web3-eth-contract/src/index.js#L685
cc @cmcewen